src/main/java/camp/woowak/lab/coupon/domain/CouponValidator.java

```diff
@@ -0,0 +1,50 @@
+package camp.woowak.lab.coupon.domain;
+
+import java.time.LocalDateTime;
+
+import camp.woowak.lab.coupon.exception.InvalidCreationCouponException;
+
+public class CouponValidator {
+	private static final int MAX_TITLE_LENGTH = 100;
+	private static final int MIN_DISCOUNT_AMOUNT = 0;
+	private static final int MIN_QUANTITY = 0;
+
+	private CouponValidator() {
+	}
+
+	public static void validate(String title, int discountAmount, int quantity, LocalDateTime expiredAt) {
+		validateTitle(title);
+		validateDiscountAmount(discountAmount);
+		validateQuantity(quantity);
+		validateExpiredDate(expiredAt);
+	}
+
+	private static void validateTitle(String title) {
+		if (title == null || title.isEmpty()) {
```

Comment:
title은 공백이 되면 안되기 때문에 isEmpty보단 isBlank로 변경하는것이 좋을 것 같습니다!
```suggestion
		if (title == null || title.isBlank()) {
```

Reply:
[b6b1724](https://github.com/woowa-techcamp-2024/Team1-3K1K/pull/71/commits/b6b17240e3a47f7c6921d23415d085062bf85342) 에 반영했습니다! 감사합니다